src/use-query.spec.ts

```diff
@@ -532,6 +532,43 @@ describe('useQuery', () => {
       expect(query).toHaveBeenCalledTimes(2)
     })
 
+    it('does not refresh/refetch when disabled', async () => {
```

Comment:
```suggestion
    it('does not automatically refresh when enabled is false', async () => {
```

I think it's better to explicitly use the name of the option for code search puposes